SpywareFilter/sections/tracking_servers.txt

```diff
@@ -40,6 +40,7 @@
 !||i.snssdk.com/log/*
 !||i.snssdk.com/slardar/sdk.js
 !
+||analytics.appsbnet.com.br
```

Comment:
@fabianski7
Please move to adservers_firstparty

Reply:
thanks for the review, but the added domain appears to be collecting browsing data and not displaying ads. It also seems to me that there is no adservers_firstparty in the SpanishFilter folder

Reply:
@fabianski7
Sorry, I got confused. I meant tracking_servers_firstparty.txt in the SpywareFilter =)